src/main/java/trello/entities/Card.java

```diff
@@ -19,7 +19,7 @@
  */
 public class Card {
     private String id;
```

Comment:
Please, add white space above this line.

Reply:
Solved